snafu/stressng_wrapper/stressng_wrapper.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+
+import os
+import argparse
+
+from .trigger_stressng import Trigger_stressng
+
+
+class stressng_wrapper():
+
+    def __init__(self, parent_parser):
+        parser_object = argparse.ArgumentParser(description="StressNG Wrapper script",
+                                                parents=[parent_parser],
+                                                formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+        parser = parser_object.add_argument_group("StressNG benchmark")
+        parser.add_argument(
+            '-u', '--uuid', nargs=1,
+            help='Provide the uuid')
+        parser.add_argument(
+            '-j', '--jobfile',
+            help='Provide the jobfile for stressNG', required=True)
+        parser.add_argument(
+            'h', '--help',
+            help='Usage: stressng_wrapper.py -j <jobfile>', required=False)
+        self.args = parser.parse_args()
```

Comment:
I have no idea how CI passed. jobfile is not being passed by https://github.com/cloud-bulldozer/benchmark-operator/pull/412
These lines need some fixes:
```suggestion
        self.args = parser_object.parse_args()
```